### PR TITLE
fix: protable search&reset button i18n

### PR DESCRIPTION
**Pull request by liuxiran**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [x] Bugfix
___
### Bugfix
- Description

I found that the `Search` and `Reset` button did not change with the locale

2020-12-30 11-27-11屏幕截图

2020-12-30 11-26-51屏幕截图

- How to fix?
Protable search config `searchText` and `resetText` accept  `string` value [1], so I just tried to use `${formatMessage({ id: 'component.global.search' })}` , and it worked well

Reference:
[1] https://procomponents.ant.design/components/table/#search-%E6%90%9C%E7%B4%A2%E8%A1%A8%E5%8D%95

### CODE REVIEW

File: web/src/pages/Consumer/List.tsx [hunk 1]
```diff
@@ -22,12 +22,12 @@ import { history, useIntl } from 'umi';
 import { PlusOutlined } from '@ant-design/icons';
 
 import { timestampToLocaleString } from '@/helpers';
-import useForceIntl from '@/hooks/useForceIntl';
+// import useForceIntl from '@/hooks/useForceIntl';
```
Comment (LiteSun) [inline web/src/pages/Consumer/List.tsx:25]: please remove the useless comment.

**Comment by LiteSun** [comment]
I check the branch code and found a problem blow. Please have a check when you free. @liuxiran 

image

**Comment by juzhiyuan** [comment]
> I check the branch code and found a problem blow. Please have a check when you free. @liuxiran
> 
> image

I just fixed it in #1157

File: web/src/pages/Consumer/List.tsx (hunk 1 above)
Comment (liuxiran) [reply web/src/pages/Consumer/List.tsx]: fixed

**Comment by liuxiran** [comment]
> I check the branch code and found a problem blow. Please have a check when you free. @liuxiran
> 
> image

good catch, already fixed, missed this module

**Comment by liuxiran** [comment]
> > I check the branch code and found a problem blow. Please have a check when you free. @liuxiran
> > image
> 
> I just fixed it in #1157

How lucy we are to solve the same issue^_^,  just another way to solve the problem, only for your reference @juzhiyuan

**Review by juzhiyuan — APPROVED** [review]

File: web/src/pages/SSL/List.tsx [hunk 2]
```diff
@@ -109,8 +107,8 @@ const Page: React.FC = () => {
         actionRef={tableRef}
         request={fetchList}
         search={{
-          searchText: formatMessage({ id: 'component.global.search' }),
-          resetText: formatMessage({ id: 'component.global.reset' }),
+          searchText: `${formatMessage({ id: 'component.global.search' })}`,
+          resetText: `${formatMessage({ id: 'component.global.reset' })}`,
```
Comment (juzhiyuan) [inline web/src/pages/SSL/List.tsx:111]: why not use formatMessage({ id: 'component.global.reset' }) directly?

Comment (liuxiran) [reply web/src/pages/SSL/List.tsx:111]: yes. you are right. 
Infact when I debuged this feature, use `formatMessage({ id: 'component.global.reset' })` directly could not worked well, so I used thie way instead, and I could not figured out why. But now it worked, any way, fixed

**Review by LiteSun — APPROVED** [review]
LGTM!

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**